pkg/cmd/cli/describe/deployments.go

```diff
@@ -105,6 +105,9 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 				deployment = &item
 			}
 		}
+		if deployment == nil {
```

Comment:
@kargakis i think this little fix should do, but I create https://github.com/openshift/origin/issues/11010 to investigate.